.cspell.json

```diff
@@ -0,0 +1,18 @@
+{
+  "version": "0.1",
+  "language": "en",
+  "ignorePaths": [
+    "**/node_modules/**",
+    "**/pkg/**",
+    "**/dist/**",
+    "bundle-size-stats.json",
+    "target/**",
+    "**/.cache/**",
+    "**/dist/**",
+    "**/.git/**",
+    "package-lock.json",
+    "yarn.lock",
+    "**/yarn-error.log"
+  ],
```

Comment:
This is the config file for cspell. In the "words" section you define words that should be treated as known by the spell checker.